samples/client/petstore/python/petstore_api/api_client.py

```diff
@@ -116,7 +113,7 @@ def __call_api(self, resource_path, method,
                                                     collection_formats)
             for k, v in path_params:
                 resource_path = resource_path.replace(
-                    '{%s}' % k, quote(str(v), safe=''))  # no safe chars, encode everything
+                    '{%s}' % k, quote(str(v)))
```

Comment:
should I also revert other unrelated changes in regenerated samples? e.g. `fake_classname_tags123_api.py` renamed to `fake_classname_tags_123_api`